.github/ISSUE_TEMPLATE/package.yml

```
@@ -0,0 +1,63 @@
+name: Packages
+description: Report problems and suggest ideas for the packages
+labels: [package]
+
+body:
+  - type: markdown
+    attributes:
+      value: >-
+        Before opening a new issue, make sure it isn't covered by an existing issue.
+        Please search for [issues with the `package` label][package-issues] (including
+        closed issues) before you continue.
+
+
+        [package-issues]: https://github.com/simple-icons/simple-icons/issues?q=is%3Aissue+label%3Apackage+is%3Aopen
+
+  - type: dropdown
+    attributes:
+      label: Kind of issue
+      description: >-
+        If your issue type is not here, select "other" and explain in the
+        "Description" field below.
+      options: [Improvement, Bug, Other]
```

Thinking this should maybe have a "Feature (request)" option

```suggestion
      options: [Bug, Feature, Improvement, Other]
```

In which case, maybe "Improvement" should be replaced by "Change" to have a clearer distinction.

```suggestion
      options: [Bug, Feature, Change, Other]
```

In either of these cases, the description of the "Description" input should probably have the following change:

```diff
- For improvements: An example of a use case
+ For feature requests: An example of a use case
```

---

What about just "Feature," "Bug," and "Other." IMO a change is not needed unless it adds a feature or fixes a bug and if there truly is something else, it can go under "Other."

---

That is a fair point, let's do those three then. You could add "Performance" as well, but not sure if that really applies here... (Though part of our discussion on #6189 suggests it could happen).

Still need to update the description of the "Description" field in that case.

---

I'd put #6189 under feature requests. I don't think that necessarily improves the performance of the package, but it does improve the developer experience. 

I do see the case in which someone has a request related only to performance like, "the package has some data that could be removed" so adding it sounds good to me.

We can update the description to change improvements to feature requests, and for performance, perhaps add this: "For performance, a clear description of the issue or how performance can be improved."

---

To clarify, I was referring specifically to [this comment](https://github.com/simple-icons/simple-icons/pull/6189#pullrequestreview-730099074), if we hadn't realized this before merging that could've lead to an issue regarding performance.

> We can update the description to change improvements to feature requests, and for performance, perhaps add this: "For performance, a clear description of the issue or how performance can be improved."

Agree with the changes. I would suggest adding something like "For performance: An example where performance is poor" instead. First, the description should always be clear so putting that only for performance feels a bit odd. Second, asking the reporter how to improve the performance seems somewhat pointless: if they did enough research to know how to improve performance they'll definitely share it; if they did not do that, they probably don't have any (good) suggestions either. That said, feel free to include that as part of the description if you disagree.

---

Your wording makes more sense and is in line with the wording for the other issue types, so I agree.